modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving high availability components
+
+HA is available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
+
+[id="serving-disabling-HA_{context}"]
+== Disabling high availability
+
+You can disable HA for these components by changing the configuration of the `KnativeServing.spec.highAvailability` field.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed the {ServerlessOperatorName} and Knative Serving.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName}*.
+. Click *Knative Serving* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Serving* tab.
++
+image::serverless-installed-operator.png[Installed Operators page]
+. Click on `knative-serving` in the *Knative Serving* tab.
+. Click on `YAML` in the *knative-serving* page.
++
+image::serving-YAML-HA.png[Knative Serving YAML]
+. In the `spec` section of the YAML, you will see the `high-availability` field configured with `2` replicas:
++
+[source,yaml]
+----
+  high-availability:
+    replicas: 2
+----
++
+Change the value of `replicas` to `1`:
```

Comment:
Consider making this it's own step since it's actionable. Or consider condensing the step into something like:

```
. In the `spec` section of the YAML, update the `high-availability.replicas` field to `1`:
+
[source,yaml]
----
  high-availability:
    replicas: 1
----
```